keps/sig-scheduling/1923-try-nominated-node-first/README.md

```diff
@@ -0,0 +1,157 @@
+- [Implementation History](#implementation-history)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [x] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [ ] (R) Graduation criteria is in place
+- [ ] (R) Production readiness review completed
+- [ ] Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+If the incoming pod that going to be scheduled has the priority defined,
+and there is no available nodes in the cluster match with the needs for
+the pod to be scheduled, scheduler will try to preempt lower priority pods
+running on each of the nodes in the cluster to release enough room for the
+pod to be scheduled, the selected node for the incoming pod after the
+preemption will be set as the value of `pod.Status.NominatedNodeName`.
+The nominated node here implies the node that is selected for the incoming
+pod to be scheduled.
+
+Pod scheduling is split into two phases, the scheduling cycle and the binding
+cycle, the scheduling cycle primarily includes predicate and scroing.
+
+When the preemption is happened, the phase of predicate for the nominated node
+has been properly measured, there is a high chance the selected nominated node
+is still the *only* one candidate for the incoming pod to be scheduled.
+
+This KEP proposes to validate the availability of the nominated node directly
+in the new scheduling cycle before looping all over the nodes in the cluster
+for the phase of the predicate.
+
+## Motivation
+
+In real production environment, pods can have different priorites due to business
+needs, the preemption could happen to make sure high priority pods could got scheduled
+and spawned.
+
+For the cluster with huge number of computing nodes, looping over all the nodes in
+the cluster for the phase of the predicate is an unnegligible consuming for scheduler,
+besides, it's unpredicatable that how the out-of-tree plugins are defined, whether
```

Comment:
we don't need to mention out-of-tree plugins, the previous phrase should be enough.